File: Java/1511A-Review Site.java
import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int sum = 0;
		
		for(int i = 0; i < n; i++){
		    int m = sc.nextInt();
		    for(int j = 0; j < m; j++){
		        int tmp = sc.nextInt();
		        if(tmp==1 || tmp ==3){
		            sum += 1;
		        }
		    }
		    System.out.println(sum);
		    sum = 0;
		}
	}
}

File: C++/1511A-Review Site.cpp
#include <iostream>
using namespace std;

int main(){
    int n, m, tmp, sum=0;
    cin>>n;
    for(int i = 0; i < n; i++){
        cin>>m;
        for(int j = 0; j < m; j++){
            cin>>tmp;
            if(tmp==1 || tmp==3){
                sum += 1;
            }
        }
        cout<<sum<<endl;
        sum = 0;
    }
    return 0;
}
